packages/desktop-client/src/components/budget/misc.js

```diff
@@ -396,6 +398,12 @@ function SidebarCategory({
                   onDelete(category.id);
                 } else if (type === 'toggleVisibility') {
                   onSave({ ...category, hidden: !category.hidden });
+                } else if (type === 'apply-single-category-template') {
+                  let month = monthUtils.currentMonth();
```

Comment:
I could use a pointer on how this should operate.  If you're on a single month view and looking at THIS month, it appears to work correctly.  If you're on a single month view and on a DIFFERENT month, it will not appear to work correctly.  If you're on a multi-month view🤷‍♂️